src/tls.c

```diff
@@ -50,6 +50,31 @@
     #include "libntruencrypt/ntru_crypto.h"
     #include <wolfssl/wolfcrypt/random.h>
 #endif
+#ifdef HAVE_LIBOQS
```

Comment:
I'd prefer this in documentation somewhere, but not sure where is a good place. This is okay for now.

Reply:
Yeah, I had the same notion so I thought about putting it in the README file. but when I tried to add it there I felt like it was too prominent.  :shrug: So, I just took your advice and put it here. Maybe one day we can have a more generic INSTALL file and have guidance on all the different integrations.

Reply:
I just noticed that there IS an INSTALL file.  Would it be okay if I moved this stuff over there?

Reply:
Yes go for it... "I just noticed that there IS an INSTALL file. Would it be okay if I moved this stuff over there?"